css/properties/scroll-snap-coordinate.json

```diff
@@ -0,0 +1,62 @@
+{
+  "css": {
+    "properties": {
+      "scroll-snap-coordinate": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/scroll-snap-coordinate",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "39"
+            },
+            "firefox_android": {
+              "version_added": "39",
+              "flag": {
+                "type": "preference",
+                "name": "layout.css.vertical-text.enabled",
+                "value_to_set": "true"
+              }
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": "abandoned",
```

Comment:
I'm not sure why we allow strings here. I think it should be `false` here and below.

Reply:
Oh, sorry I was unclear. I meant false (because it is abandoned so no more experiment).

But you are right the schema should have catched this.